src/logger.js

```diff
@@ -57,33 +63,61 @@ function buildLogger(config) {
       winston.format.timestamp(),
       winston.format.errors({ stack: true }),
     ),
-    defaultMeta: { service: 'wallet' },
+    defaultMeta: { service: defaultService },
     transports,
   });
 
+  return logger;
+}
+
+/**
+ * Build an application logger instance.
+ *
+ * @param {Configuration} config
+ * @return {winston.Logger}
+ */
+function buildAppLogger(config) {
+  if (logger) {
+    return logger;
+  }
+
+  const appLogger = buildLogger(config, 'wallet');
+
   // create a stream object with a 'write' function that will be used by `morgan`
-  logger.stream = {
-    write(message, encoding) {
+  appLogger.stream = {
+    write(message, _encoding) {
       // use the 'info' log level so the output will be picked up by
       // both transports (file + console)
-      logger.info(message.trim(), {
+      appLogger.info(message.trim(), {
         service: 'http',
       });
     },
   };
+  /* eslint-disable no-console */
+  console.log = (...args) => appLogger.info.call(appLogger, ...args);
+  console.info = (...args) => appLogger.info.call(appLogger, ...args);
+  console.warn = (...args) => appLogger.warn.call(appLogger, ...args);
+  console.error = (...args) => appLogger.error.call(appLogger, ...args);
+  console.debug = (...args) => appLogger.debug.call(appLogger, ...args);
+  /* eslint-enable no-console */
 
-  console.log = (...args) => logger.info.call(logger, ...args);
-  console.info = (...args) => logger.info.call(logger, ...args);
-  console.warn = (...args) => logger.warn.call(logger, ...args);
-  console.error = (...args) => logger.error.call(logger, ...args);
-  console.debug = (...args) => logger.debug.call(logger, ...args);
-
-  return logger;
+  return appLogger;
 }
 
 // This fixes some logs where Github code scanning complains about a `Log Injection` possibility
-const sanitizeLogInput = input => String(input).replace(/\n|\r/g, '');
+function sanitizeLogInput(input) {
+  return String(input).replace(/\n|\r/g, '');
+}
```

Comment:
Why did you change this?

Reply:
We were having errors on the tests because `sanitizeLogInput` was not a function.
I believe that the issue was on how the exports were being organized, since we added more exports in the file it changed the behavior.

The method is the same, only that now it is a function.